config/initializers/shut_up_sass.rb

```diff
@@ -0,0 +1,15 @@
+# https://github.com/rails/sass-rails/issues/242
```

Comment:
This is unfortunate - but otherwise we get a bunch of warnings from bootstrap because of quirks in the Twitter Bootstrap gem.  This causes the warnings to go away.  I just copied and pasted from the GitHub issue.